defaults/main.yml

```diff
@@ -425,3 +472,64 @@ galaxy_systemd_env: []
 
 # A list of additional python packages to install into galaxy's virtual environment
 galaxy_additional_venv_packages: []
+
+# galaxy themes variables
+galaxy_themes_subdomains: {}
+# - name: assembly
+#   tool_sections:
+#     - "hicexplorer"
+#     - "graph_display_data"
+#     - "peak_calling"
+#     - "assembly"
+#     - "annotation"
+#     - "genome_diversity"
+#     - "multiple_alignments"
+#   extra_tool_labels:
+#     - "proteomics"
+galaxy_themes_static_keys:
+  static_dir: ""
+  static_images_dir: "images/"
+  static_scripts_dir: "scripts/"
+  static_welcome_html: "welcome.html/"
```

Comment:
```suggestion
  static_welcome_html: "welcome.html"
```

Reply:
i guess it makes sense to have these as vars even if I don't love them being overridden

Reply:
`welcome.html` is actually a directory

Reply:
D: what

Reply:
wat